Python/tier2.c

```diff
@@ -14,7 +14,7 @@
 // Max typed version basic blocks per basic block
 #define MAX_BB_VERSIONS 10
 
-#define OVERALLOCATE_FACTOR 7
+#define OVERALLOCATE_FACTOR 10
```

Comment:
Note: need to increase the overallocate factor because we now have 10 instruction long cache entries for all the BB stuff...